.eslintrc

```diff
@@ -0,0 +1,158 @@
+rules:
+  indent:
+    - 2
+    - 2
+  quotes:
+    - 2
+    - single
+  linebreak-style:
+    - 2
+    - unix
+  semi:
+    - 2
+    - never
+  array-bracket-spacing: 
+    - 2
+    - "never"
+  block-scoped-var: 2
+  brace-style: 
+    - 2
+    - "stroustrup"
+    - 
+      allowSingleLine: true
+  comma-dangle: 
+    - 2
+    - "never"
```

Comment:
@tizzo Sounds like from what you wrote, the dangling comma rule should be "always", but it's still "never" in the repo.